src/revivalpmmp/pureentities/components/MobEquipment.php

```diff
@@ -39,7 +39,6 @@
 use pocketmine\item\IronHelmet;
 use pocketmine\item\IronLeggings;
 use pocketmine\item\Item;
-use pocketmine\item\ItemIds;
```

Comment:
Changing the use of ItemIds to Item is not required to be compatible with 3.19.0.  Please revert all related changes and open a separate PR to handle this.